src/App.js

```diff
@@ -8,38 +7,40 @@ import Footer from "./components/common/Footer";
 import Home from "./components/home/Home";
 import StaffList from "./components/staff/StaffList";
 import PetsList from "./components/pets/PetsList";
-
-/*
-  Data
-  ---------------
-  Note: Normally this data would be pulled from an API. It is not necessary, however, for this application.
-*/
 import { employees } from "./data/employees.js";
 import { owners } from "./data/owners";
 import { pets } from "./data/pets";
 
+import { Switch, Route, Link } from "react-router-dom";
+import PetsListNav from "./components/pets/PetsListNav";
 class App extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      employees,
-      owners,
-      pets,
-    };
-  }
-
   render() {
-    const { employees, owners, pets } = this.state;
     return (
       <div className="wrapper">
-        <Nav />
+        {/* <Nav />
         <Home employees={employees} owners={owners} pets={pets} />
         <StaffList employees={employees} />
         <PetsList pets={pets} />
-        <Footer />
+        <Footer /> */}
+        <Switch>
```

Comment:
Since the Nav and footer is always rendering you can have the component called outside the switch statement and won't have to add it to each route. 

```
return (
  <div>
    <nav/>
    <switch>
       ...
    </switch>
    <footer/>
  </div>
)
```